tests/baselines/reference/castExpressionParentheses.js

```diff
@@ -74,12 +74,12 @@ a;
 a[0];
 a.b["0"];
 a().x;
+1..foo;
+1..foo;
+1.0.foo;
+12e+34.foo;
+0xff.foo;
 // should keep the parentheses in emit
-(1).foo;
-(1.).foo;
-(1.0).foo;

```

Comment:
Maybe it's just me, but I actually prefer the old emit since it's closer to what I'd think a human would write. Is there any way we can continue doing the parenthesization?